### ISSUE TITLE
Cannot use it: problem with dependencies from core-js

### ISSUE THREAD

**Issue by alexmoreno** [AlbertBrand/vue-async-function#4]
``` ERROR  Failed to compile with 15 errors                                                                                        15:09:12

These dependencies were not found:

* core-js/modules/es6.array.filter in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.array.for-each in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.array.is-array in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.array.iterator in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.date.to-string in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.object.define-properties in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.object.define-property in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.object.keys in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.object.to-string in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.promise in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.regexp.to-string in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es6.symbol in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es7.object.get-own-property-descriptors in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/es7.symbol.async-iterator in ./node_modules/vue-async-function/dist/index.js
* core-js/modules/web.dom.iterable in ./node_modules/vue-async-function/dist/index.js
```

Already tried to install `core-js` with `npm` and `yarn`, `2.6.10`, `2.6.5`... same error.

**Comment by AlbertBrand** [comment]
It's currently fixed on core-js 2.x, so make sure to install core-js 2.x in your project (the default for Vue projects created with the CLI). I can have a look if you share a repo with your code.

**Comment by AlbertBrand** [comment]
Ah, it seems that with vue-cli v4 the default is now core-js3. I'll update the lib to fix this.

**Comment by AlbertBrand** [comment]
Fixed in release 3.0.0. corejs3 is now an explicit peer dependency.
